runtests.py

```diff
@@ -27,10 +27,10 @@ def run_tests(args, settings_module):
 
     # Run all tests except Selenium tests using SQLite
     sqlite_args = args.copy()
-    sqlite_args.extend(['--exclude-tag', 'selenium_tests'])
+    sqlite_args.extend(['--exclude-tag', 'selenium_tests', '--parallel', '1'])
     run_tests(sqlite_args, settings_module='openwisp2.settings')
 
     # Run Selenium tests using PostgreSQL
     psql_args = args.copy()
-    psql_args.extend(['--tag', 'selenium_tests'])
+    psql_args.extend(['--tag', 'selenium_tests', '--parallel', '1'])
```

Comment:
Please remove the parallel flag here, we add this from the command line when launching tests. 

What is forcing you to add it here?